capture-sdk/sdk/src/main/res/layout-sw600dp/gc_fragment_camera.xml

```diff
@@ -361,6 +361,19 @@
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 
+    <include
```

Comment:
[Request] Please align the notification to the frame as shown in the tablet design (sorry for forgetting to show you beforehand where the tablet designs are 😞):
- portrait: https://www.figma.com/design/LJT8L5kpqe00Zbb2NDK55B/Gini-Photo-Payment-%2F-Android?node-id=4363-25341&t=kmLMeePHJd0GY4h1-0